tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -723,7 +736,11 @@ protected void updateViewport() {
   protected void onScrollChanged( int l, int t, int oldl, int oldt ) {
     super.onScrollChanged( l, t, oldl, oldt );
     updateViewport();
-    requestThrottledRender();
+    if( mShouldRenderWhilePanning ) {
+      requestRender();
```

Comment:
This is done in purpose, because if we call `requestSafeRender`, the fling will cause the call of `requestThrottledRender`, and we do not want this.
Even during a fling, there is no stutter while rendering.